levels/c1.md

```diff
@@ -27,3 +27,5 @@ Works on all parts of the [codebase](https://github.com/moka-care/levels/blob/gu
 ### Data
 
 Anticipates /presents data structure changes implications
+Organize taks planning by priority
```

Comment:
```suggestion
Organize tasks planning by priority
```

Reply:
There is a notion of Theme / Reserve Owner in C1 that can never be filled by someone in Data currently, so _by construction_ nobody can go above 😬  (sadly)  

Maybe we should change / add something there too? Like Theme / Reserve / Data Owner ?